src/basic/architecture.c

```diff
@@ -66,6 +66,9 @@ int uname_architecture(void) {
 #elif defined(__ia64__)
                 { "ia64",       ARCHITECTURE_IA64     },
 
+#elif defined(__loongarch64)
+                { "loongarch64", ARCHITECTURE_LOONGARCH64 },
+
```

Comment:
`l` is before `m`. Please move this before `m68k`.

Also, please drop the dot from the commit message :pray:

Reply:
Done.